aptos-move/framework/aptos-framework/sources/timestamp.move

```diff
@@ -45,8 +45,8 @@ module aptos_framework::timestamp {
         } else {
             // Normal block. Time must advance
             assert!(now < timestamp, error::invalid_argument(EINVALID_TIMESTAMP));
+            global_timer.microseconds = timestamp;
         };
-        global_timer.microseconds = timestamp;
```

Comment:
This is unnecessary but is technically a no-op if (proposer == @vm_reserved) so more of a cleanup? :)

Reply:
save gas? lol.

Reply:
I'd not worried about gas for this one line. It's trivial.

Reply:
I am kidding tho